### PR TITLE
Add scrobble info for main user to README 

### PR DESCRIPTION
**Pull request by agneevX**

### CODE REVIEW

**Comment by agneevX** [comment]
Yeah sorry please ignore that.

**Comment by agneevX** [comment]
> remove note that username filter is not supported

I actually added instructions that they are in fact supported because they aren't documented anywhere else.

**Comment by glensc** [comment]
> > remove note that username filter is not supported
> 
> I actually added instructions that they are in fact supported because they aren't documented anywhere else.

yes, because I asked the user who requested to do that.

File: README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ docker-compose run --rm plex-trakt-sync
 To run `watch` command:
 
 ```
-docker-compose run --rm plex-trakt-sync watch
+docker-compose run plex-trakt-sync watch
```
[the remark this answers is not in the document]
Comment (glensc) [reply README.md:141]: `--rm` flag has nothing to do with this. read the docker-compose documentation.

and you probably want to have `-d` here instead, and in any case that should be a separate pull request with explanation why, etc.

**Comment by agneevX** [comment]
I hope someone else can add the suggestions I made above.

**Pull request closed without merging**